tests/framework/microvm.py

```diff
@@ -373,6 +373,17 @@ def dimensions(self):
             "guest_memory": f"{self.mem_size_bytes / (1024 * 1024)}MB",
         }
 
+    @property
+    def guest_kernel_version(self):
+        """Get the guest kernel version from the filename
+
+        It won't work if the file name does not like name-X.Y.Z
+        """
+        splits = self.kernel_file.name.split("-")
+        if len(splits) < 2:
+            return None
```

Comment:
nit: Could we add an error message here, so that we make debugging later on easier?

Reply:
Yeah I think a warning could be warranted. I will add it in a follow-up.